### DISCUSSION TITLE
Some images of the final version.

### DISCUSSION THREAD

**Discussion by GlancingMind** [GlancingMind/git-bug#43] [Show and tell]
![image](https://user-images.githubusercontent.com/9444609/113147452-13501400-9231-11eb-9870-90c88d912d51.png)
---
![image](https://user-images.githubusercontent.com/9444609/113147919-92dde300-9231-11eb-87b9-ce79e1d162a8.png)
---
Profile page of user without any bugs
![image](https://user-images.githubusercontent.com/9444609/113148088-bf91fa80-9231-11eb-80a6-b318397bc426.png)
Profile page of user with authored bugs
![image](https://user-images.githubusercontent.com/9444609/113148266-f49e4d00-9231-11eb-9918-43179ba840db.png)

**Comment by GlancingMind** [comment]
@MichaelMure @claudioantonio (@LenaNoctua @AienTech @timbkr) I thought you might want to see some pictures. I will gradually upstream the remaining changes.

**Reply by MichaelMure** [reply]
There is always ways to nitpick and improve, but that's really cool. Great work!

On a side note, isn't the markdown rendering broken? This should work in the webUI IIRC.

**Reply by GlancingMind** [reply]
Yeah I noticed that too. In the release version (7.1) it worked.
But when we took up work, the dev webui didn't render markdown.
(We started of from this commit https://github.com/GlancingMind/git-bug/tree/upstream)

Anyway, it was a quick fix. :) https://github.com/GlancingMind/git-bug/pull/44
Unfortunately I couldn't incorporate a fix for https://github.com/MichaelMure/git-bug/issues/580
But at least it supports emojis :partying_face: https://github.com/MichaelMure/git-bug/issues/579#issuecomment-811382776
